README.md

```diff
@@ -29,22 +29,19 @@ npm install @moxy/next-sitemaps
 **1. Add the plugin to `next.config.js`**
 
 ```js
-const withPlugins = require('next-compose-plugins');
 const withSitemap = require('@moxy/next-sitemaps/plugin');
 
-withPlugins([
-    // other plugins..
-    withSitemap(),
-    // other plugins...
-]);
+module.exports = (phase) => withSitemap(phase, 'https://moxy.studio')({ ...nextConfig });
```

Comment:
Aren't we missing the `nextConfig` as an argument to the export, next to `phase`?

Reply:
The `nextConfig` is the first argument of the returned function.